src/App.js

```diff
@@ -34,7 +35,11 @@ const configValues = {
 setConfig(configValues);
 
 function App() {
-  return <div className="bc_services-list-app"></div>;
+  return (
+    <div>
```

Comment:
shouldn't need the div here anymore, I think this was just being used to store that "app" class.

Reply:
removed div